src/main/java/Domain/Game/Team.java

```diff
@@ -98,16 +100,19 @@ public boolean addTeamManager(TeamOwner townr,Role teamManager)
         return false;
     }
 
-    public boolean addTeamOwner(TeamOwner townr,TeamOwner teamOwner)
+    public boolean addTeamOwner(TeamOwner teamOwner)
     {
-        if(!teamOwners.contains(townr))
-        {
-            return false;
-        }
 
         return teamOwners.add(teamOwner);
```

Comment:
have you tested what will happen if teamOwner is null?

Reply:
👍